src/Libraries/AssertNet.Moq/Assertions.cs

```diff
@@ -14,6 +14,16 @@ public static class Assertions
     /// <param name="mock">Mock under test.</param>
     /// <typeparam name="T">Type of the mock.</typeparam>
     /// <returns>Assertion about a mock.</returns>
+    public static MockAssertion<T> That<T>(this AssertionBuilder _, Mock<T> mock) where T : class
```

Comment:
Perhaps it would be better if this extension method was placed in a seperate static class that resides in the `AssertNet` namespace to ensure it is always imported?

Reply:
I'll do that.